### ISSUE TITLE
User must make many attempts to actually log out, intermittently

### ISSUE THREAD

**Issue by dllh** [Automattic/wp-calypso#76099]
### Quick summary

TL;DR: User frequently has to try several (or even many) times to log out and sees either blank pages or errors when trying, until after repetition, it finally works.

This is a weird, intermittent bug that a user is encountering. I cannot personally reproduce, nor have any HEs been able to. There is a javascript error I wanted to note in case it was relevant. I don't think [this PR](https://github.com/Automattic/wp-calypso/pull/73855) is relevant, but it's the only fairly recent thing I could find that referenced the javascript property that's sending an error to the user's browser.

See 6080495-zd-woothemes.

### Steps to reproduce

I added the console steps that begin and end this repro list, to see if the user could spot anything useful in the console that might point to a server-side issue, as this has occurred on multiple browsers and devices with cookies/etc. cleared.

1. Load the browser console.
2. Click the user avatar at the top right of the admin bar from any screen. This should go to /me.
3. Observe that a blank screen loads rather than the profile.
5. Click back to get to the prior page.
6. See step 3. Keep trying this until:
7. The message "We're sorry but an unexpected error has occurred" appears in the main content area to the right of the usual admin sidebar.
8. Look for any errors in the browser console (my debugging step for the user).

### What you expected to happen

The user is logged out and sees an appropriate logout screen.

### What actually happened

The user reports:

> When I try to log out, the page My Profile appears, over and over again. Sometimes have to try three or four times to log out.

The user also reported that sometimes it takes many more tries (like a couple of dozen). And sometimes the logout works as expected, so it's intermittent.

Further, when they sent me info from the browser console, this one came up:

![Screen Shot 2023-04-21 at 3 56 46 PM](https://user-images.githubusercontent.com/2738252/233723900-b12be44e-3fdf-4e6b-a7db-fc99cef4fa84.png)

It may not be related to this at all (I sort of doubt it is, in fact), but a javascript error seemed worth reporting in any case.

### Impact

One

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple

### Logs or notes

The user has cleared cache and cookies on multiple browsers and computers. They have also seen this behavior on their phone. I don't think we can consider this a browser-specific issue, hence my suggesting we look for console errors that _could_ point to some other (maybe server-side?) problem. I don't actually think the error sent back actually points to such an issue.

The user says he can live with the issue but wanted to bring it to our attention. This seems like it must be some kind of edge case, as I could find no other similar reports. So there may not be much to do here, but perhaps the js error is worth a look.

**Comment by cuemarie** [comment]
Thank you for the detailed notes on this while excluding sensitive user details, @dllh ! We'll see what we can do to get the right eyes on this

**Comment by cuemarie** [comment]
@mmtr I know you're likely not working on stuff like this at the moment, I just found that you helped resolve a log-out issue a while back (https://github.com/Automattic/wp-calypso/issues/51662) and thought I'd ask you for some direction help! Do you have any insight into who might be able to help look into this javascript error this user's dealing with? Anything you recognize?

**Comment by mmtr** [comment]
Thanks for the ping @cuemarie. The JS error reported in the summary seems related to Staging sites (pesfPa-2E-p2), so maybe someone from @Automattic/yolo can help?

**Comment by wojtekn** [comment]
@mmtr, thanks for the ping, we are checking it.

**Comment by dllh** [comment]
Just noting that the user wrote back to share that they had looked at console output in Edge (vs. Chrome). They provided console output, which showed the same output as the screen shot in the original report here.

**Comment by wojtekn** [comment]
@dllh the issue should be fixed now.

**Comment by dllh** [comment]
@wojtekn thank you for a very speedy resolution! I've let the customer know to try again.
